src/components/notes/saving-indicator.jsx

```diff
@@ -0,0 +1,83 @@
+import React, { useRef, useState } from 'react'
+import { translate } from 'cozy-ui/react/I18n'
+import { Text } from 'cozy-ui/react/Text'
+
+import styles from 'components/notes/saving-indicator.styl'
+
+// constants
+const sec = 1000
+const min = 60 * sec
+const hour = 60 * min
+const day = 24 * hour
+
+// differents steps for message formating
+// [age, key, unit, coef] :
+// - `age`: the first list where the age of last save
+// if younger than this `age` will be used
```

Comment:
I don't understand what you mean here. What is "the first list"? Do you mean that for a given last save time, the first item for which `age < now - last save time` is true will be used?

Reply:
I rephrased

Reply:
Thanks